content/blog/scales-1-3-0/index.Rmd

```diff
@@ -0,0 +1,101 @@
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+```
+We're delighted to announce the release of [scales](https://scales.r-lib.org) 1.3.0. scales is a packages that extracts much of the scaling logic that is used in ggplot2 to a general framework, along with utility functions for e.g. formatting labels or creating color palettes.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+This blog post will give a quick overview of the 1.3.0 release, which is mainly an upkeep release but does contain a few interesting tidbits.
+
+You can see a full list of changes in the [release notes](%7B%20github_release%20%7D)
+
+```{r setup}
+library(scales)
+set.seed(1)
+```
+
+## Proper support for difftime objects
+
+While scales have had rudimentary support for objects from the hms package it has not included basic support for difftime objects from base. This is now rectified with the introduction of `label_timespan()`, `breaks_timespan()`, and `transform_timespan()`. While the labels and breaks function can be used on their own, all the behavior is encapsulated in the timespan transform object which is kin to `transform_hms()`.
+
+```{r}
+library(ggplot2)
+
+events <- data.frame(
+  time = runif(30, max = 200),
+  magnitude = rnorm(30) + 2
+)
+
+ggplot(events) + 
+  geom_point(aes(time, y = 0, size = magnitude), position = position_jitter(height = 0)) + 
+  scale_x_continuous(trans = transform_timespan(), limits = c(0, NA))
+```
+
+As we can see the timespan transform by default understand numeric data as seconds elapsed. Further it identifies that for this range, adding breaks for minutes makes most sense.
+
+If we had recorded time as hours rather than seconds, we can inform the transformer of this:
+
+```{r}
+ggplot(events) + 
+  geom_point(aes(time, y = 0, size = magnitude), position = position_jitter(height = 0)) + 
+  scale_x_continuous(trans = transform_timespan("hours"), limits = c(0, NA))
+```
+
+## API brush-up
+
+scales has gone through a number of touch-ups on its API, latest with a revamp of the labels functions which are now all prefixed with `label_`. With this release we continues (and hopefully concludes) this by changing the naming of the transformation utilities and the palettes. More in line with how we cater to tab-completion when designing functions they all now shares a `transform_` and `pal_` prefix respectively. Further, we have moved from the word `trans` to the more descriptive `transform` throughout the API. On top of that we have renamed the `label_dollar()` function to `label_currency()` to make it clear that this can be used for any type of currency, not just US dollars. All the old functions have been kept around with no plan of deprecation but we advice you to update your code to use the new names.
```

Comment:
```suggestion
scales has gone through a number of touch-ups on its API, such as revamping the labels functions to all start with `label_`. This release we continue (and hopefully conclude) the touch-ups by using a common prefix for the transformation utilities (`transform_`) and palettes (`pal_`). We have also rename `label_dollar()` to `label_currency()` to make it clear that this can be used for any type of currency, not just dollars (US or otherwise). All the old functions have been kept around with no plan of deprecation but we advise you to update your code to use the new names.
```

Reply:
Maybe say they've been superseded since that's the official term.